.gitignore

```
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```

אני לא בטוח מאיפה לקחת את הקובץ הזה אבל אמור להיות פה גם את ה env. הרגיל
```suggestion
.env.production.local
.env
```

---

הוספתי למרות שזה לא עשה לי בעיות בריצה